harris_county_bookings/jims_recorder.py

```diff
@@ -0,0 +1,79 @@
+import csv
+import io
+import os
+
+import requests
+
+from constants import *
+from github_publisher import GitHubPublisher
+from utils import *
+
+
+class JIMSRecorder(object):
+    # The headers left out of this list are the headers that contain personal data
+    # https://github.com/open-austin/harris-county-bookings/issues/4
+    HEADERS_TO_SAVE = ['SEX', 'RACE', 'ARREST DATE', 'BOOKING DATE', 'ADDRESS NUMBER', 'ADDRESS PREFIX',
+                       'ADDRESS STREET', 'ADDRESS SUFFIX', 'ADDRESS ALI', 'ADDRESS CITY', 'ADDRESS STATE',
+                       'ADDRESS ZIP', 'CHARGE CODE', 'CHARGE WORDING', 'CHARGE LEVEL', 'DISPOSITION']
+
+    ACCDB = 'accdb'
+    CSV = 'excel'
+    ALL_DIALECTS = [CSV, ACCDB]
+    csv.register_dialect(ACCDB, delimiter=';')
+
+    @staticmethod
+    def read():
+        return requests.get(JIMS_1058_URL).text
+
+    @staticmethod
+    def parse():
+        raw_data = JIMSRecorder.read().split('\n')
+        headers = raw_data[0].split(';')
+        data = list(csv.DictReader(raw_data[1:], fieldnames=headers, dialect=JIMSRecorder.ACCDB))
```

Comment:
`JIMSRecorder.ACCDB` seems very indirect.

Because I first saw `accb` out of the context of this line, I had to google what `ACCDB` means.

Had I seen it on this line, I could have inferred it is a CSV dialect.